scripts/create-user.sh

```diff
@@ -138,19 +140,67 @@ create_org_space() {
 }
 
 create_user() {
-  if [[ "${RESET_USER}" == "true" ]]; then
-    if cf delete-user "${EMAIL}" -f 2>&1 | tee "${TMP_OUTPUT}"; then
-      if grep -q "does not exist" "${TMP_OUTPUT}"; then
-        abort "Trying to reset password for non-existing user. Is someone trying to trick you into getting an account?"
-      fi
+  export INVITE_URL
+  local uaa_endpoint auth_token ssl_arg uaa_uuid
+
+  uaa_endpoint=$(cf curl /v2/info | jq -er '.authorization_endpoint')
+  auth_token=$(cf oauth-token)
+  ssl_arg=$(if jq -e '.SSLDisabled == true' ~/.cf/config.json >/dev/null; then echo "-k"; fi)
+
+  curl -sf "${ssl_arg}" \
+    -H "Authorization: ${auth_token}" \
+    -H "Accept: application/json" -H "Content-Type: application/json" \
+    -G --data-urlencode "filter=userName eq \"${EMAIL}\"" \
+    "${uaa_endpoint}/Users" >"${TMP_OUTPUT}"
+
+  if jq -e '.resources | length > 1' "${TMP_OUTPUT}" >/dev/null; then
+    cat "${TMP_OUTPUT}"
+    abort "Multiple UAA users found for: ${EMAIL}"
+  fi
+
+  if jq -e '.resources | length == 1' "${TMP_OUTPUT}" >/dev/null; then
+    uaa_uuid=$(jq -er '.resources[0].id' "${TMP_OUTPUT}")
+
+    if [[ "${RESET_USER}" == "true" ]]; then
+      curl -sf "${ssl_arg}" \
+        -X PATCH \
+        -H "Authorization: ${auth_token}" \
+        -H "Accept: application/json" -H "Content-Type: application/json" \
+        -H "If-Match: *" \
+        -d "{\"verified\": false}" \
```

Comment:
Had a chat with @dcarley offline about this. Setting it to `false` allows the invite mechanism to work for existing users.